frontend/src/views/MovieAddScreen.vue

```diff
@@ -121,7 +121,7 @@ export default {
         if (this.image) {
           const data = new FormData();
           data.append('image', this.image);
-          await axios.post(`/api/cinema/movies-${movie.id}-upload-image`, data, imageConfig);
+          await axios.post(`/api/cinema/movies/${movie.id}/upload-image/`, data, imageConfig);
```

Comment:
Try to replace the following line
```suggestion
          await axios.post(`${import.meta.env.VITE_API_URL}/api/cinema/movies/${movie.id}/upload-image/`, data, imageConfig);
```